src/components/insights/PulseCard.svelte

```diff
@@ -97,8 +99,9 @@ mixin rawHtml(variables)
 
   .text {
     @include text('caption');
-    color: var(--waterloo);
     margin: 10px 0 12px;
+    font-size: 16px;
+    font-weight: 400;
```

Comment:
`@include text('caption');` on line 101 should be changed to `@include text('body-2');`